main.rb

```diff
@@ -0,0 +1,46 @@
+require 'open3'
+
+def env_has_key(key)
+	return (ENV[key] != nil && ENV[key] !="") ? ENV[key] : abort("Missing #{key}.")
+end
+
+$output_path = env_has_key("AC_OUTPUT_DIR")
+$repo_path = env_has_key("AC_REPOSITORY_DIR")
+$npm_params = (ENV["AC_RN_TEST_COMMAND_ARGS"] != nil && ENV["AC_RN_TEST_COMMAND_ARGS"] !="") ? ENV["AC_RN_TEST_COMMAND_ARGS"] : nil
+
+def run_command(command)
+  puts "@@[command] #{command}"
+  stdout_str, stderr_str, status = Open3.capture3(command)
+
+  if status.success?
+    puts stdout_str unless stdout_str.empty?
+    puts stderr_str unless stderr_str.empty?
+    return stderr_str.empty? ? stdout_str : stderr_str
+  else
+    return stderr_str
+  end
+end
+
+def runTests
+  
+  yarn_or_npm = File.file?("#{$repo_path}/yarn.lock") ? "yarn" : "npm"
+  
+  report_command = $npm_params.nil? ? "jest --coverage --coverageDirectory='coverage' --coverageReporters='lcov'" : $npm_params
```

Comment:
I realised 2 problem:
1. There is wrong logic here. When arg is entered, the command is skipped.

For example when I pass "--debug" as arg then result:
![Screenshot 2024-10-31 at 13 27 41](https://github.com/user-attachments/assets/36d1d119-d8e4-49b9-82c5-a762b4edbffb)


2. And IMO if there is no test result it must give an warn message